### PR TITLE
Wishlist options across website

### PR DESCRIPTION
**Pull request by ScoobyDubeyDo**
-Added the current wishlist I the page.
- add to wishlist button
- remove from wishlist button

### CODE REVIEW

File: src/pages/components/CartWishCard/CartWishCard.jsx [hunk 1]
```diff
@@ -1,37 +1,42 @@
...
                 />
                 <div className="card-side-content">
-                    <h2 className="card-title text-noWrap">{title}</h2>
+                    <h2 className="card-title text-noWrap">{name}</h2>
                     <p className="card-subtitle rootShoot-full-width">
-                        {prodTypes}
+                        {type.toLocaleString()}
                     </p>
                     <p className="card-title heading-6 rootShoot-full-width">
                         {price}
```
Comment (Pratik1005) [inline src/pages/components/CartWishCard/CartWishCard.jsx:33]: Just a suggestion: you can add the currency symbol before price

**Review by Pratik1005 — APPROVED** [review]
UI looking good, wish list is working properly just a suggestion added. 👍

File: src/context/wishlist/wishlist-context.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            try {
+                const res = await axios.get("/api/user/wishlist", {
+                    headers: { authorization: localStorage.getItem("token") },
+                });
+                setWishlist(res.data.wishlist);
+            } catch (err) {
+                console.log(err.message, "error while getting wishlist");
+            }
+        })();
+        console.log(currentUser, "user");
```
Comment (manismk) [inline src/context/wishlist/wishlist-context.jsx:21]: can remove the console logs

**Review by manismk — APPROVED** [review]
Code is clean and easy to understand.

Bug: If I click add to wishlist quickly duplicate items are added to the wishlist. since it's an async call, you can disable the button till it was added.
image

**Pull request merged**